docs/scaling/README.md

```diff
@@ -21,3 +21,11 @@ autoscaling.knative.dev/maxScale: "10"
 ```
 
 You can also use these annotations directly on `kpa` objects.
+
+**NOTE**: These annotations apply for the full lifetime of a `revision`.
+Even when a `revision` is not referenced by any `route`, the minimal pod count
+specified by `autoscaling.knative.dev/minScale` will still be provided. Non-routed
```

Comment:
From this [comment](https://github.com/knative/serving/issues/2720#issuecomment-458250775), there's reasons to keep current behavior.

I added the description that these annotations are not subject to Conformance in case we make the change in future.